libvirt/resource_cloud_init.go

```diff
@@ -31,6 +31,7 @@ func resourceCloudInit() *schema.Resource {
 			},
 			"user_data": &schema.Schema{
 				Type:     schema.TypeString,
+				Default:  "#cloud-config\nssh_authorized_keys: []\n",
```

Comment:
@monstermunchkin  for let the test passing, we need to add a default.
The reason is that, we are creating dynamically  this variables after a `terraform plan` and then this are stored in terraform.states.
if we reapply a `terraform plan`, it reforce a resource creation, because this are empty by default.

if you do 2 times ``terraform apply`` with the same ile 

```hcl
resource "libvirt_cloudinit" "commoninit" {
  name = "commoninit.iso"
  local_hostname = "node"
}
```
it forces you to recreate the resource, even it shouldn't

```console

erraform will perform the following actions:

-/+ libvirt_cloudinit.commoninit (new resource required)
      id:             "/var/lib/libvirt/images/commoninit.iso;5a0d652a-6738-1eb6-cf14-f15a5a564ad9" => <computed> (forces new resource)
      local_hostname: "node" => "node"
      name:           "commoninit.iso" => "commoninit.iso"
      pool:           "default" => "default"
      user_data:      "#cloud-config\nssh_authorized_keys: []\n" => "" (forces new resource)


Plan: 1 to add, 0 to change, 1 to destroy.
````